src/App.tsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function isInclude(word: string, part: string) {
+  return word.toLowerCase().includes(part.toLowerCase().trim());
+}
+
+function getFilteredList(movieList: Movie[], keyWord: string) {
+  const visibleMovies = movieList.filter(movie => (
+    isInclude(movie.title, keyWord) || isInclude(movie.description, keyWord)
+  ));
+
+  return visibleMovies;
+}
+
 export const App: React.FC = () => {
+  const [query, setStateQuery] = useState('');
+
+  const handlerChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleQuaryChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364